### ISSUE TITLE
Why am I not able to copy snippets of code from this book?

### ISSUE THREAD

**Issue by pnelligan** [DruidMech/UE4-Book-Repo#1]
The code is not formatted in this book (kindle version), and what's worse, every copy of the text made comes with a signature. This is a bad oversight and makes navigating the book a nightmare. If I want to copy and paste lines of text, I should be able to! ... Surely you don't want me to type every line of code??!!

**Comment by DruidMech** [comment]
Hello,
I was not aware of this issue. Thank you for bringing this to my attention.
I will see what I can do to fix this. In the meantime, please download the
Unreal Engine project from my GitHub site:
GitHub.com/DruidMech/UE4-Book-Repo. All the code is there.
Again, I apologize for the inconvenience.
Stephen

**Comment by pnelligan** [comment]
Ok, apologies for my tone there. Was a little frustrated by the clunkiness. Otherwise enjoying the book and it seems well structured. FYI, I'm working off the Kindle app for Mac, and copying from there. Seems the signature is automatic, but maybe formatting as code (or something else) can get around this? Also, the code copied from the book is not at all formatted, so line breaks etc. need to be put in manually. Also may be some issue with hidden characters, but didn't confirm.

**Comment by DruidMech** [comment]
No, I totally understand. My intention was for the code to be copyable
directly from the Kindle book, so I'll need to format a PDF version (the
paperback version was a formatted PDF). So I appreciate you letting me know
that this was not the case.
I'm glad you're enjoying the book otherwise! More content is in the making.
My Best,
Stephen
